pkgs/test/lib/src/runner/browser/default_settings.dart

```diff
@@ -24,7 +24,7 @@ final defaultSettings = UnmodifiableMapView({
   ),
   Runtime.firefox: ExecutableSettings(
       linuxExecutable: 'firefox',
-      macOSExecutable: '/Applications/Firefox.app/Contents/MacOS/firefox-bin',
+      macOSExecutable: '/Applications/Firefox.app/Contents/MacOS/firefox',
```

Comment:
Was this a change in how firefox installs itself, should we support both paths?

Reply:
Yes, this is an effect of the change in where Firefox is installed.

https://github.com/mozilla/gecko-dev/commit/9d7f71a3cecd09998b81d653db5ec42aab464d0a
https://bugzilla.mozilla.org/show_bug.cgi?id=1871447

---

Should it be modified to support two paths? It doesn't seem easy and I haven't come up with a way to support it...

Reply:
Yes we should modify it to support two paths.

Note that you can use a local (or even global?) configuration file to set this path in the meantime to work around things.

Reply:
I pushed a quick fix for this